api/versioning/BUILD

```diff
@@ -31,7 +31,7 @@ proto_library(
         "//contrib/envoy/extensions/filters/network/sip_proxy/tra/v3alpha:pkg",
         "//contrib/envoy/extensions/filters/network/sip_proxy/v3alpha:pkg",
         "//contrib/envoy/extensions/matching/input_matchers/hyperscan/v3alpha:pkg",
-        "//contrib/envoy/extensions/network/connection_balance/dlb/v3alpha:pkg",
+        "//contrib/envoy/extensions/dlb/v3alpha:pkg",
```

Comment:
This and the above are weird I think. I think @phlax was suggesting to move the extension source to top-level, not the API proto, which for the most part mirrors the directory structure of the regular Envoy API tree, see https://github.com/envoyproxy/envoy/tree/main/api/contrib/envoy/extensions.

Reply:
I know.